stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -306,7 +308,9 @@ public class ChatClient internal constructor(
                 )
             }
 
-            override fun cancel() {}
+            override fun cancel() {
+                // no-op
```

Comment:
What about replacing this empty block with:
```
override fun cancel() = Unit
```
Does it also produce an error?

Reply:
Let me check it